### PR TITLE
ConfigLess update

### PR DESCRIPTION
**Pull request by ManojBudumuru-vungle**
This commit update adapter to remove the set for init completion calbacks.

IOS-7033

### CODE REVIEW

File: Adapters/Vungle/ISVungleAdapter/ISVungleBannerDelegate.m [hunk 1]
```diff
@@ -39,9 +39,7 @@ - (void)bannerAdDidFail:(VungleBannerView * _Nonnull)bannerView
     NSError *bannerError = [NSError errorWithDomain:kAdapterName
                                                code:errorCode
                                            userInfo:@{NSLocalizedDescriptionKey:error.description}];
-    if (self.isAdloadSuccess) {
-        [self.delegate adapterBannerDidFailToShowWithError:bannerError];
```
Comment (ashinagawa) [inline Adapters/Vungle/ISVungleAdapter/ISVungleBannerDelegate.m:43]: We don't need this call back?

**Review by ashinagawa — APPROVED** [review]
Change looks good.
Sanity check OK.

File: Adapters/Vungle/ISVungleAdapter/ISVungleBannerDelegate.m (hunk 1 above)
Comment (ManojBudumuru-vungle) [reply Adapters/Vungle/ISVungleAdapter/ISVungleBannerDelegate.m:43]: @ashinagawa its giving build errors and they no longer have this in their class. So i removed it.

Comment (ashinagawa) [reply Adapters/Vungle/ISVungleAdapter/ISVungleBannerDelegate.m:43]: I see. got it. Thanks for clarifying it.

**Review by YueVungle — APPROVED** [review]
LGTM 👍

File: Adapters/Vungle/ISVungleAdapter/ISVungleAdapter.m [hunk 2]
```diff
@@ -13,10 +13,9 @@
 #import <VungleAdsSDK/VungleAdsSDK.h>
 
 // Handle init callback for all adapter instances
-static ISConcurrentMutableSet<ISNetworkInitCallbackProtocol> *initCallbackDelegates = nil;
 static InitState initState = INIT_STATE_NONE;
```
Comment (stanley-vungle) [inline Adapters/Vungle/ISVungleAdapter/ISVungleAdapter.m]: You should remove this. Even if it failed in the adapter, the SDK can maybe get it in the ads response

Comment (stanley-vungle) [reply Adapters/Vungle/ISVungleAdapter/ISVungleAdapter.m]: You can also remove all the switch statements that uses this

**Review by crbinsf — APPROVED** [review]
Changes look good.

**Review by stanley-vungle — DISMISSED** [review]
Hold on this PR for a bit. I'm refactoring the adapter to remove the delegate mapping for the inits. It doesn't need it with the config less feature.

**Review by bobVungle — APPROVED** [review]
LGTM

**Review by YueVungle — APPROVED** [review]
LGTM 👍

**Review by stanley-vungle — APPROVED** [review]

**Pull request merged**